### PR TITLE
Proposed fix for Safety Module

### PR DESCRIPTION
**Pull request by def-main**
Instructions for deployment and tests are in [contracts/safety/README.md](contracts/safety/README.md)

### CODE REVIEW

File: src/deploy-config/base-config.ts [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    '0x688134aefae00632d36754ca9f085bd25072e2f0',
+    '0xb50309682298fecf08ea07792154fa26f05fd7e5',
+    '0x49fae0a92272ec68583eb009f4c40bcf91b9010f',
+    '0xad2955994fb189117478eec154153ba7d0f835ca',
+  ],
+
+  // Safety Module recovery.
+  //
+  // Compensation amount is 10% of what was initially staked and stuck in the Safey Module.
+  SM_RECOVERY_COMPENSATION_AMOUNT: '15745887438533773204745', // About 15746 DYDX.
```
Comment (lucas-dydx) [inline src/deploy-config/base-config.ts:168]: Super small nit: Include comment on how this was calculated + what date we snapshotted the funds at?`157,458.87438533773204764 * 0.1 = 15,745.8874385338` (using [safety module balance on Etherscan]( https://etherscan.io/token/0x92d6c1e31e14520e676a687f0a93788b716beff5?a=0x65f7BA4Ec257AF7c55fd5854E5f6356bBd0fb8EC)).

Comment (Kenadia) [reply src/deploy-config/base-config.ts:173]: Updated comments

Comment (def-main) [reply src/deploy-config/base-config.ts:173]: Updated comments

**Comment by noursale** [comment]
How I make stk

**Pull request merged**